.storybook/head.html

```diff
@@ -0,0 +1,12 @@
+<script type="text/javascript" src="//fast.fonts.net/jsapi/f7f47a40-b25b-44ee-9f9c-cfdfc8bb2741.js"></script>
+<script type="text/javascript" src="https://fast.fonts.net/jsapi/core/mt_otf.js"></script>
+
+<style>
+  @font-face {
+	font-family: 'artsy-icons';
+	src: url("/fonts/artsy-icons.eot?uo9ko");
+	src: url("/fonts/artsy-icons.eot?#iefixuo9ko") format('embedded-opentype'), url("/fonts/artsy-icons.woff2?uo9ko") format('woff2'), url("/fonts/artsy-icons.ttf?uo9ko") format('truetype'), url("/fonts/artsy-icons.woff?uo9ko") format('woff'), url("/fonts/artsy-icons.svg?uo9ko#artsy-icons") format('svg');
+	font-weight: normal;
+	font-style: normal;
+}
+</style>
```

Comment:
Yeah would make sense to have a module for global css instead using this syntax

```javascript
import { injectGlobal } from 'styled-components'

injectGlobal`
    @font-face {
      font-family: 'Operator Mono';
      src: url('../fonts/Operator-Mono.ttf');
    }
`
```

and import it from any module that needs it. I'll set it up.